src/main/java/org/opentripplanner/standalone/config/buildconfig/S3BucketConfig.java

```diff
@@ -32,24 +32,15 @@ public static S3BucketConfig fromConfig(NodeAdapter config) {
     }
     S3BucketConfig bucketConfig = new S3BucketConfig();
     try {
-      bucketConfig.accessKey =
-        config
-          .of("accessKey")
-          .withDoc(NA, /*TODO DOC*/"TODO")
-          .withExample(/*TODO DOC*/"TODO")
-          .asString();
-      bucketConfig.secretKey =
-        config
-          .of("secretKey")
-          .withDoc(NA, /*TODO DOC*/"TODO")
-          .withExample(/*TODO DOC*/"TODO")
-          .asString();
-      bucketConfig.bucketName =
-        config
-          .of("bucketName")
-          .withDoc(NA, /*TODO DOC*/"TODO")
-          .withExample(/*TODO DOC*/"TODO")
-          .asString();
+      /*TODO DOC*/
+      /*TODO DOC*/
```

Comment:
```suggestion
```

Reply:
These are fixed in a later PR, so I would prefer just to merge to avoid conflicts.